src/client/src/app/components/login/login.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+
```

Comment:
Please add loginForm as FormGroup and Formbuilder in the constructor to describe the username and password are required